colossus/src/main/scala/colossus/service/ServiceDSL.scala

```diff
@@ -185,9 +151,30 @@ trait ClientLifter[C <: Protocol, T[M[_]] <: Sender[C,M]] {
 trait ClientFactory[C <: Protocol, M[_], T <: Sender[C,M], E] {
 
 
+  /**
+    * Load a ServiceClient definition from a Config.  Looks into `colossus.clients.$clientName` and falls back onto
+    * `colossus.client-defaults`
+    * @param clientName The name of the client definition to load
+    * @param config A config object which contains at the least a `colossus.clients.$clientName` and a `colossus.client-defaults`
+    * @return
+    */
+  def apply(clientName : String, config : Config = ConfigFactory.load())(implicit provider: ClientCodecProvider[C], env: E) : T = {
```

Comment:
I think it might be better to create a `ClientConfig.Default` and have `config`default to that, so that we're not loading config all the time.  That's what we do for `ServiceConfig`.